docker/irods_client/tests/test_sync.py

```diff
@@ -7,31 +7,23 @@ def test_sync_dry_run(session, testdata, capsys):
 
     ipath = IrodsPath(session, "~", "empty")
     coll = create_collection(session=session, coll_path=ipath)
+    assert len(coll.data_objects)+len(coll.subcollections)==0, "Dry run starting not empty"
 
     # upload
-    sync_data(session=session,
-         source=testdata,
-         target=ipath,
-         max_level=None,
-         dry_run=True,
-         copy_empty_folders=True)
-
-    captured = capsys.readouterr()
-    lines=sorted([x.strip() for x in captured.out.split("\n") if len(x.strip())>0])
-
-    assert lines == ['/tempZone/home/rods/empty/more_data',
-        'Will create collection(s):',
-        "Will upload from '/tmp/testdata' to '/tempZone/home/rods/empty':",
-        'beach.rtf  727',
-        'bunny.rtf  10150',
-        'example.r  182',
-        'more_data/polarbear.txt  2717',
-        'plant.rtf  992',
-        'plant.rtf.copy  992',
-        'sun.rtf  661']
+    ops = sync_data(session=session,
+                    source=testdata,
+                    target=ipath,
+                    max_level=None,
+                    dry_run=True,
+                    copy_empty_folders=True)
 
+    assert len(ops["create_collection"]) == 1
```

Comment:
I think we should replace that with a normal check. As @dometto  found in another PR, these asserts can get confused with unit and integration test asserts.

Reply:
This is in a test file though no? 😄